site/content/internal/writing-a-blog-post.md

```diff
@@ -5,11 +5,56 @@ section: internal
 weight: 105
 ---
 
-Went to a conference recently? Worked on something cool? Got something else Mattermost-related you want to post about? Follow these steps to write a blog post:
+Been to a conference recently? Worked on something cool? Got something else Mattermost-related you want to post about? Writing a blog post is a great way to share your experience with the community. 
 
-1. Clone https://github.com/mattermost/mattermost-developer-documentation 
-2. Create a new .md file in the [/site/content/blog/](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/blog) folder
-  - Use `YYYY-MM-DD-<your-blog-post-title>.md` as the filename
+Blog posts can cover a wide range of topics, such as:
+
+- Addressing a customer-facing problem
+- Describing an experience with Mattermost/your Mattermost implementation 
```

Comment:
```suggestion
- Describing an experience with Mattermost / your Mattermost implementation 
```

Reply:
I generally don't use a space when using a slash but I know in our documentation we have instances where we do and instances where we don't. Grammatically it's meant to be the latter but happy to agree on one, standardize it, and then apply across the board. Thoughts?

Reply:
We can leave as is for now here.